HonestHouse/HonestHouse/Sources/Core/Network/DTO/API+Namespace.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-enum ShootingControl { }
-enum ShootingSettings { }
-enum LiveView { }
-enum ImageOperations { }
+enum ImageOperations { }    // 4.7. Image Operations
+enum ShootingControl { }    // 4.8. Shooting Control
+enum ShootingSettings { }   // 4.9. Shooting Settings
+enum CameraStatus { }       // 4.13. Camera Status
```

Comment:
/// 으로 주석처리하면 팝업에 설명이 떠요 ...!
```suggestion
enum ImageOperations { }    /// 4.7. Image Operations
enum ShootingControl { }    /// 4.8. Shooting Control
enum ShootingSettings { }   /// 4.9. Shooting Settings
enum CameraStatus { }       /// 4.13. Camera Status
```